.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "workbench.colorCustomizations": {
+        "activityBar.background": "#213030",
+        "titleBar.activeBackground": "#2E4444",
+        "titleBar.activeForeground": "#F9FBFB"
+    }
+}
```

Comment:
can we add this to `.gitignore` so not everyone has to accept these settings, please?

Reply:
yes - that one slipped through!